metaci/plan/migrations/0013_create_default_plans.py

```diff
@@ -87,6 +87,15 @@ def create_default_plans(apps, schema_editor):
         public=False,
     )
 
+    Plan.objects.create(
+        name="Performance Test",
+        description=("Runs the test_performance_scratch flow aginst a scratch org"),
+        type="manual",
+        flows="test_performance_scratch",
+        org="dev",
+        public=False,
+    )
```

Comment:
Is the goal here to create a Performance Test plan for all new instances of MetaCI only? Or to add one to all existing instances of MetaCI when they upgrade?

Should this use the long-running queue?

Reply:
New instances and tests. And yes.

Reply:
bc7fe79db